airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/cursor/file_based_cursor.py

```diff
@@ -32,7 +32,7 @@ def set_initial_state(self, value: StreamState) -> None:
         """
 
     @abstractmethod
-    def get_state(self) -> Mapping[str, Any]:
+    def get_state(self) -> MutableMapping[str, Any]:
```

Comment:
more out of curiosity, but is there a reason why to use MutableMapping vs a Dict in general? does mypy care?

Reply:
It only matters because the `IncrementalMixin` [defines the state as a MutableMapping](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/core.py#L54)